docs/topics/data_calculation.md

```diff
@@ -0,0 +1,105 @@
+# How to calculate data values
+
+Other topics in this same document have addressed how to insert data in the Platform and how to query and modify that
+data. But, what is the best way to make automatic calculations based on those data values? There are different ways of
+calculating data based on its origin and on the nature of the calculation.
+
+## Calculations based on data coming from the devices
+
+For those pieces of data coming from measurement reports from the Devices, the IoTAgents offer mechanisms to make calculations
```

Comment:
"IoTAgents" could be a link to the .md in this documentation describing IOTA (device gateway).